idd/Energy+.idd.in

```diff
@@ -61186,19 +61274,37 @@ NodeList,
   A492,A493,A494,A495,A496,A497,A498,A499,A500,A501; \note fields as indicated
 
 OutdoorAir:Node,
-       \memo This object sets the temperature and humidity conditions
-       \memo for an outdoor air node.  It allows the height above ground to be
-       \memo specified.  This object may be used more than once.
-       \memo The same node name may not appear in both an OutdoorAir:Node object and
-       \memo an OutdoorAir:NodeList object.
+      \memo This object sets the temperature and humidity conditions
+	  \memo for an outdoor air node.  It allows the height above ground to be
+	  \memo specified.  This object may be used more than once.
+	  \memo The same node name may not appear in both an OutdoorAir:Node object and
+	  \memo an OutdoorAir:NodeList object.
+	  \memo This object defines local outdoor air environmental conditions.
   A1,  \field Name
        \required-field
        \type node
-  N1 ; \field Height Above Ground
+  N1,  \field Height Above Ground
        \note A value less than zero indicates that the height will be ignored and the weather file conditions will be used.
        \type real
        \units m
        \default -1.0
+  A2 , \field Drybulb Temperature Schedule Name
+       \type object-list
+       \object-list ScheduleNames
+       \note Schedule values are real numbers, -100.0 to 100.0, units C 
+  A3 , \field Wetbulb Schedule Name
```

Comment:
We found that in the struct NodeData {} which is used for outdoor air node. These variables are already defined.

```
Real64 OutAirDryBulb; // {C}
bool EMSOverrideOutAirDryBulb; // if true, the EMS is calling to override outdoor air node drybulb setting
Real64 EMSValueForOutAirDryBulb; // value EMS is directing to use for outdoor air node's drybulb {C}
Real64 OutAirWetBulb; // {C}
bool EMSOverrideOutAirWetBulb; // if true, the EMS is calling to override outdoor air node wetbulb setting
Real64 EMSValueForOutAirWetBulb; // value EMS is directing to use for outdoor air node's wetbulb {C}
```

...and used for the calculation of Enthalpy, HumRat at the node level. We were thinking it would be more straightforward for implementation in this way.